rust-toolchain

```diff
@@ -1 +0,0 @@
-nightly-2018-10-14
```

Comment:
Could you instead just replace it with `nightly` so that people don't have to switch to nightly manually, while at the same time not pinning to a specific nightly?

Reply:
They will still have to switch to `nightly` manually when doing the other steps; this override only controls compilation of Miri. This will be especially confusing if they have their default compiler set to a specific nightly, which is common for people using unstable compiler tools.